SimPEG/EM/NSEM/SurveyNSEM.py

```diff
@@ -64,27 +83,31 @@ def eval(self, f):
     def evalDeriv(self, f):
         raise Exception('Use Sources to project fields deriv.')
 
-#################
+#########
 # Data
-#################
+#########
 
 
-class Data(SimPEGsurvey.Data):
-    '''
+class Data(SimPEGsurvey.Data, DataNSEM_plot_functions):
```

Comment:
That sounds fine with me. In regards to the `Data` classes, I actually think it might be nicer to have `ImpedanceData`, `TipperData`, etc classes and use `Combo` objective when one is jointly inverting. Makes interacting simpler and more effective. I also think that is something `properties` are ideal for.